mlxtend/feature_selection/sequential_feature_selector.py

```diff
@@ -201,6 +207,13 @@ def __init__(self, estimator, k_features=1,
         self.verbose = verbose
         self.clone_estimator = clone_estimator
 
+        if not isinstance(early_stop_rounds, int) or early_stop_rounds < 0:
+            raise ValueError('Number of early stopping round should be '
+                             'an integer value greater than or equal to 0.'
+                             'Got %d' % early_stop_rounds)
```

Comment:
I think `...Got %d' % early_stop_rounds` might not work if `early_stop_rounds` is not an integer. Maybe it's better to replace it with

`...Got %s' % early_stop_rounds`

Reply:
Right!